db/change_cache.go

```diff
@@ -309,7 +309,7 @@ func (c *changeCache) waitForSequenceWithMissing(sequence uint64) {
 
 // Given a newly changed document (received from the tap feed), adds change entries to channels.
 // The JSON must be the raw document from the bucket, with the metadata and all.
-func (c *changeCache) DocChanged(docID string, docJSON []byte, seq uint64, vbNo uint16) {
+func (c *changeCache) DocChanged(docID string, docJSON []byte, seq uint64, vbNo uint16, dataType uint8) {
```

Comment:
Couldn't each bit flag in the collection of bit flag be considered a uint constant?  And then the aliased type would ensure that a valid constant was used?  (as opposed to any arbitrary uint)